client/sites-dashboard/components/sites-site-launch-nag.tsx

```diff
@@ -71,6 +71,10 @@ const getChecklistSlug = ( site: SiteExcerptData ) => {
 		return 'legacy-site-setup';
 	}
 
+	if ( intent === 'assembler-first' ) {
+		return 'legacy-site-setup';
+	}
```

Comment:
Required for old test sites I had with an assembler-first intent set, without this it was showing 5/7 complete in the indicator but 1/5 in my home tasks.

Reply:
This looks kind of wrong but the `/home` checklist is using the `legacy-site-setup` intent when calling the `/launchpad` endpoint so this should be ok.

Reply:
@p-jackson do you know if there's a fallback launchpad tasks for unsupported intents? Basically assembler-first and design-first are kept for backwards compatibility, but I'd like to remove them.

Reply:
I think this happens here~ish: https://github.com/Automattic/wp-calypso/blob/update/checklist-display/client/my-sites/customer-home/cards/launchpad/site-setup.tsx#L4